File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

//extends: herencia 
class UberBlack extends Car {
    //Eso representa la marca y la coleccion de 2 datos representara el modelo
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    //clase constructor
    public UberBlack(String license, Account driver, 
    Map<String, Map<String,Integer>> typeCarAccepted,
    ArrayList<String> seatsMaterial){
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}

File: Python/UberBlack.py
from car import Car

class UberBlack(Car):
    #Array vacio
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial
